### ISSUE TITLE
training results of alpaca-lora 7b  and original llama 7b

### ISSUE THREAD

**Issue by frankzhao112** [tloen/alpaca-lora#406]
![image](https://user-images.githubusercontent.com/128885107/235035086-b6b6f89b-0f49-41f4-bb57-8f87a206a129.png)
1 first we load both llama 7b and aplaca-lora-7b .
![image](https://user-images.githubusercontent.com/128885107/235038742-1ad7a715-b6e4-4ab7-ab99-a55a4ffe7f31.png)
2 u can use colab to test both llama -7b and aplaca-lora-7b ， u can not find any improvements at all. Using int8 type and peft lora training llama significantly reduces the cost of gpu memory , but the training results can not satisfy the needs of a online chat system.

3 after I use other datasets to  train llama using  int8 type and peft lora traning, I still get the same results.

**Comment by BinWang28** [comment]
I tried to fine-tune with my own dataset. Similarly, it seems not learning much. Should I adjust the learning rate or parameters for LORA? Any advice?
